### PR TITLE
13.14 kaggle dog

### PR DESCRIPTION
**Pull request by goldmermaid**
Forum links

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-807/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

**Comment by mli** [comment]
Job d2l-zh/PR-807/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

**Comment by mli** [comment]
Job d2l-zh/PR-807/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

**Comment by goldmermaid** [comment]
`numref`, `cite` check done

File: chapter_computer-vision/kaggle-dog.md [hunk 1]
```diff
@@ -79,46 +83,45 @@ valid_ratio = 0.1
...
     torchvision.transforms.RandomResizedCrop(224, scale=(0.08, 1.0),
                                              ratio=(3.0/4.0, 4.0/3.0)),
     torchvision.transforms.RandomHorizontalFlip(),
-    # Randomly change the brightness, contrast, and saturation
+    # 随机更改亮度，对比度和饱和度
     torchvision.transforms.ColorJitter(brightness=0.4,
                                        contrast=0.4,
                                        saturation=0.4),
-    # Add random noise
+    # 添加随机噪音
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-dog.md:122]: Please replace all 噪音 with  噪声
As I suggested before, for the CV chapter you may take a look at our v1 contents before adapting it:
http://zh.d2l.ai.s3-website-us-west-2.amazonaws.com/chapter_computer-vision/kaggle-gluon-dog.html

**Comment by mli** [comment]
Job d2l-zh/PR-807/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

**Comment by mli** [comment]
Job d2l-zh/PR-807/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

**Comment by mli** [comment]
Job d2l-zh/PR-807/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-807/

File: chapter_computer-vision/kaggle-dog.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_computer-vision/kaggle-dog.md]: Fixed!

**Pull request merged**